src/components/AddItemForm.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
+import fb from '../lib/firebase';
+import '../css/AddItemForm.css';
+import { v4 as uuidv4 } from 'uuid';
+
+const Form = () => {
```

Comment:
Would we want to include the token as a prop for the form, so the data can be sent to the correct FireStore collection?

Reply:
yep! that will be added once we can pass the token from the App.js. In the meantime, the prop was sending an error in the form, so we removed it for now